genome_kit/variant_genome.py

```diff
@@ -198,10 +199,29 @@ def find_motif(self, interval, motif, match_position=0, find_overlapping_motifs=
     def __repr__(self):
         return '<VariantGenome {} with {} variants>'.format(self.reference_genome, len(self.variants))
 
+    @property
+    def reference_genome(self):
+        return self.genome.reference_genome
+
+    def __eq__(self, other):
+        if not isinstance(other, VariantGenome):
+            return False
+        if self.genome != other.genome:
+            return False
+        if self.variants != other.variants:
```

Comment:
I forget but are these normalized? I don't think so.

Depends if you believe in functional equivalence or if ordering/normalization differences matter

Reply:
Changed to functional comparison. Not crazy about using "private" apis from Variant but didn't want to make them public either.